### PR TITLE
CEDS-3159 new 'Express Consignment' page

### PR DESCRIPTION
**Pull request by udinobi**

### CODE REVIEW

File: app/views/declaration/summary/transport_section.scala.html [hunk 1]
```diff
@@ -147,6 +147,26 @@
...
+                    items = Seq(
+                        actionItem(
+                            href = controllers.declaration.routes.ExpressConsignmentController.displayPage(mode).url,
+                            content = HtmlContent(linkContent(messages("site.change"))),
+                            visuallyHiddenText = Some(messages("declaration.summary.transport.expressConsignment.change"))
+                        )
+                    )
+                ))
+            )
+        ),
```
Comment (glidester) [inline app/views/declaration/summary/transport_section.scala.html:169]: I know this is not of your doing but this whole section give me a headache trying to read it!

File: test/controllers/declaration/DepartureTransportControllerSpec.scala [hunk 2]
```diff
@@ -73,9 +73,8 @@ class DepartureTransportControllerSpec extends ControllerSpec with ErrorHandlerM
   }
 
   private def nextPage(decType: DeclarationType) = decType match {
-    case SUPPLEMENTARY => routes.BorderTransportController.displayPage()
-    case STANDARD      => routes.BorderTransportController.displayPage()
-    case CLEARANCE     => routes.TransportPaymentController.displayPage()
+    case STANDARD | SUPPLEMENTARY => routes.BorderTransportController.displayPage()
```
Comment (glidester) [inline test/controllers/declaration/DepartureTransportControllerSpec.scala:76]: :+1:

File: test/controllers/declaration/ExpressConsignmentControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        "form contains incorrect values" in {
+          val incorrectForm = Json.obj("yesNo" -> "wrong")
+
+          val result = controller.submitForm(Mode.Normal)(postRequest(incorrectForm))
+          status(result) must be(BAD_REQUEST)
+          verifyPageInvoked
+        }
+
+        "neither Yes or No have been selected on the page" in {
+          val incorrectForm = Json.obj("yesNo" -> "")
```
Comment (glidester) [inline test/controllers/declaration/ExpressConsignmentControllerSpec.scala:119]: Nit: isn't this the same test as above (incorrect value)? If the form is not selected should you not pass an empty json object (i.e. not even have the "yesNo" key either)?

**Review by glidester — APPROVED** [review]

File: test/controllers/declaration/ExpressConsignmentControllerSpec.scala (hunk 3 above)
Comment (udinobi) [reply test/controllers/declaration/ExpressConsignmentControllerSpec.scala:111]: :+1:

**Review by glidester — APPROVED** [review]

**Review by glidester — APPROVED** [review]

**Pull request merged**